src/platform/logger/journald_logger.cpp

```diff
@@ -29,6 +29,8 @@ constexpr auto to_syslog_priority(const mpl::Level& level) noexcept
 {
     switch (level)
     {
+    case mpl::Level::trace:
+        return LOG_DEBUG;
```

Comment:
Unless you're accounting for a future `LOG_TRACE` in `syslog.h`, then the `trace` and `debug` cases can be combined.

Reply:
Right! Done.